eg-next/examples/primitives-triangles.rs

```diff
@@ -1,60 +1,96 @@
 //! # Example: Triangles
 //!
-//! Shows multiple triangles with different properties
+//! Shows multiple triangles with different styles.
 
 use embedded_graphics::{
-    pixelcolor::BinaryColor,
+    pixelcolor::Rgb888,
     prelude::*,
-    primitives::{PrimitiveStyle, Triangle},
+    primitives::{PrimitiveStyle, PrimitiveStyleBuilder, StrokeAlignment, Triangle},
 };
 use embedded_graphics_simulator::{OutputSettingsBuilder, SimulatorDisplay, Window};
 
-const PAD: i32 = 10;
-
 fn main() -> Result<(), core::convert::Infallible> {
-    let mut display: SimulatorDisplay<BinaryColor> = SimulatorDisplay::new(Size::new(512, 128));
+    let mut display: SimulatorDisplay<Rgb888> = SimulatorDisplay::new(Size::new(600, 128));
 
-    let style = PrimitiveStyle::with_stroke(BinaryColor::On, 1);
+    let padding = Point::new(10, 20);
+    let size = display.bounding_box().size.height as i32 - (padding.y * 2);
+    let half_size = size / 2;
+    let offset = size + 10;
 
-    // No straight lines
-    Triangle::new(Point::new(0, 0), Point::new(64, 10), Point::new(15, 64))
-        .translate(Point::new(PAD, 0))
-        .into_styled(style)
-        .draw(&mut display)?;
+    // Triangle pointing up
+    let base_triangle = Triangle::new(
+        Point::new(0, size),
+        Point::new(half_size, 0),
+        Point::new(size, size),
+    )
+    .translate(padding);
+
+    // Triangle pointing down
+    let flipped_triangle = Triangle::new(
+        Point::new(0, 0),
+        Point::new(size, 0),
+        Point::new(half_size, size),
+    )
+    .translate(padding);
 
-    // Flat top
-    Triangle::new(Point::new(5, 0), Point::new(30, 64), Point::new(64, 0))
-        .translate(Point::new(64 + PAD, 0))
-        .into_styled(style)
+    // Inside thick stroke, no fill
+    base_triangle
+        .into_styled(
+            PrimitiveStyleBuilder::new()
+                .stroke_alignment(StrokeAlignment::Inside)
+                .stroke_width(10)
+                .stroke_color(Rgb888::CSS_SALMON)
```

Comment:
I try to follow this order in the examples:
```suggestion
            PrimitiveStyleBuilder::new()
                .stroke_color(Rgb888::CSS_SALMON)
                .stroke_width(10)
                .stroke_alignment(StrokeAlignment::Inside)
```
The reason to use this order is that the first two calls are in the same order as the arguments in `PrimitiveStyle::with_stroke`. And the alignment is an advanced option that should follow the basic settings IMO.